config.go

```diff
@@ -0,0 +1,5 @@
+package app
+
+type Config struct {
+	AllowUnstructuredContext bool `mapstructure:"allow_unstructured_context"`
```

Comment:
I'm not sure, this is an understandable configuration option name.

Reply:
true, it is a challenging part, any propositions are welcomed 

considered options
- allow_unstructured_context
- allow_context_json
- context_json_unwrap

Reply:
`context_json_unwrap` looks better 👍 
Let me ask a few PHP devs for the advice as well. It might be there is a some common name for such a thing 😃

Reply:
config was renamed